test/cpp/qps/client.h

```diff
@@ -79,7 +83,8 @@ class Client {
 
   ClientStats Mark() {
     Histogram latencies;
-    std::vector<Histogram> to_merge(threads_.size());
+    // avoid std::vector for old compilers
```

Comment:
WAT! really? no std::vector? Which c++ compiler that can be regarded as such doesn't have an STL impl with std::vector? (other than maybe not having it inside std::)